### PR TITLE
Translate api - special attributes

### PR DESCRIPTION
**Pull request by Tintef**

### CODE REVIEW

File: src/v2/api/index.md [hunk 1]
```diff
@@ -2231,115 +2231,115 @@ type: api
...
-## Special Attributes
+## Atributos Especiales
 
 ### key
 
-- **Expects:** `number | string`
+- **Espera:** `number | string`
 
-  The `key` special attribute is primarily used as a hint for Vue's virtual DOM algorithm to identify VNodes when diffing the new list of nodes against the old list. Without keys, Vue uses an algorithm that minimizes element movement and tries to patch/reuse elements of the same type in-place as much as possible. With keys, it will reorder elements based on the order change of keys, and elements with keys that are no longer present will always be removed/destroyed.
+  El atributo especial `key` es utilizado principalmente como una pista para el algoritmo del DOM virtual de Vue para identificar VNodes cuando se compara nuevas listas de nodos contra listas antiguas. Sin claves, Vue utiliza un algoritma que minimiza el movimiento de elementos e intenta parchear/reusar elementes del mismo tipo lo más posible. Con claves, este reordenará elementos basado en el orden de cambio de las claves, y los elementos con claves que ya no esten presentes, siempre serán removidos/destruidos.
```
Comment (FLarra) [inline src/v2/api/index.md]: A few typos: 

```suggestion
  El atributo especial `key` es utilizado principalmente como una pista para el algoritmo del DOM virtual de Vue para identificar _VNodes_ cuando se compara nuevas listas de nodos contra listas antiguas. Sin claves, Vue utiliza un algoritmo que minimiza el movimiento de elementos e intenta parchear/reusar elementos del mismo tipo lo más posible. Con claves, este reordenará elementos basado en el orden de cambio de las claves, y los elementos con claves que ya no están presentes, siempre serán removidos/destruidos.
```

File: src/v2/api/index.md [hunk 2]
```diff
@@ -2231,115 +2231,115 @@ type: api
...
-  When `text` changes, the `<span>` will always be replaced instead of patched, so a transition will be triggered.
+  Cuando `text` cambie, el `<span>` siempre será reemplazado en vez de parchaedo, por lo cual, una transición será lanzada.
 
 ### ref
 
-- **Expects:** `string`
+- **Espera:** `string`
 
-  `ref` is used to register a reference to an element or a child component. The reference will be registered under the parent component's `$refs` object. If used on a plain DOM element, the reference will be that element; if used on a child component, the reference will be component instance:
+  `ref` es utilizado para registrar una referencia a un elemento a un componente hijo. La referencia será registrada bajo el objeto `$refs` del componente padre. Si se utiliza en un elemento "plano" del DOM, la referncia será a dicho elemento; si se utiliza en un componente hijo, la referencia será a la instancia de dicha componente:
```
Comment (FLarra) [inline src/v2/api/index.md]: I think you are missing an `o`

```suggestion
  `ref` es utilizado para registrar una referencia a un elemento o a un componente hijo. La referencia será registrada bajo el objeto `$refs` del componente padre. Si se utiliza en un elemento "plano" del DOM, la referencia será a dicho elemento; si se utiliza en un componente hijo, la referencia será a la instancia de dicha componente:
```

File: src/v2/api/index.md [hunk 3]
```diff
@@ -2231,115 +2231,115 @@ type: api
...
+  <!-- vm.$refs.p será el nodo del DOM -->
+  <p ref="p">hola</p>
 
-  <!-- vm.$refs.child will be the child component instance -->
+  <!-- vm.$refs.child será la intancia del componente hijo -->
   <child-component ref="child"></child-component>
   ```
 
-  When used on elements/components with `v-for`, the registered reference will be an Array containing DOM nodes or component instances.
+  Cuando se utiliza con elementos/components con `v-for`, la referencia registrada será un arreglo que contendrá nodos del DOM o instancias de componentes.
```
Comment (FLarra) [inline src/v2/api/index.md]: Missing translation here:

```suggestion
  Cuando se utiliza con elementos/componentes con `v-for`, la referencia registrada será un arreglo que contendrá nodos del DOM o instancias de componentes.
```

File: src/v2/api/index.md [hunk 4]
```diff
@@ -2231,115 +2231,115 @@ type: api
...
 
-> New in 2.5.0+
+> Nuevo en 2.5.0+
 
-- **Expects:** `function argument expression`
+- **Espera:** `function argument expression`
 
 - **Uso:**
 
   Used to denote an element or component as a scoped slot. The attribute's value should be a valid JavaScript expression that can appear in the argument position of a function signature. This means in supported environments you can also use ES2015 destructuring in the expression. Serves as a replacement for [`scope`](#scope-replaced) in 2.5.0+.
```
Comment (FLarra) [inline src/v2/api/index.md]: You missed the translation of this paragraph :)

**Review by FLarra — CHANGES_REQUESTED** [review]

File: src/v2/api/index.md (hunk 4 above)
Comment (Tintef) [reply src/v2/api/index.md]: Damn 🤦‍♂

**Comment by Tintef** [comment]
@FLarra Thanks for the review! Everything should be updated 😄

**Review by FLarra — APPROVED** [review]

**Pull request merged**